.github/workflows/pythonpackage.yml

```diff
@@ -114,7 +114,7 @@ jobs:
       uses: actions/cache@v2
       with:
         path: gdcmbin
-        key: GDCM
+        key: GDCM-${{ matrix.os }}-python${{ matrix.python-version }}
 
```

Comment:
I recall now that I didn't have a specific key before (just "GDCM") because I only had it run on ubuntu-latest, and I thought the build should not be Python-version-dependent (just compiling C code I thought).  I'm curious as to why it now appears to be Python-version-dependent when it wasn't before.  It's not wrong, but there is the time penalty of building GDCM three times instead of one when the cache has expired, and those builds do take 2 to 2.5 min each.

We could leave it for now and if I remember I could have a go at it again later sometime.

Reply:
I'm wondering if perhaps it was and maybe the cache had python dependent versions in it that have only now expired. Since 3.8 gets built regularly it was the only one left.